manifests/t/TheBombermaaanteam/Bombermaaan/1.4.0.627/TheBombermaaanteam.Bombermaaan.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: TheBombermaaanteam.Bombermaaan
+PackageVersion: 1.4.0.627
+Installers:
+- InstallerUrl: https://sourceforge.net/projects/bombermaaan/files/1%20-%20Bombermaaan%20Stable/1.4/Bombermaaan_1.4.0.627_setup.exe/download
+  Architecture: x64
+  InstallerType: inno
+  InstallerSha256: 4E8BDC5362623E6F7712C7FB2442339DE3EA494FF51562F8ED86AF2239EC4745

```

```suggestion
  InstallerSha256: 4E8BDC5362623E6F7712C7FB2442339DE3EA494FF51562F8ED86AF2239EC4745
  AppsAndFeaturesEntries:
  - DisplayName: Bombermaaan 1.4
    DisplayVersion: 
    ProductCode: '{451C4ACA-0B6A-4564-BD9D-A6C365DB9C76}_is1'
  ProductCode: '{451C4ACA-0B6A-4564-BD9D-A6C365DB9C76}_is1'
```

Since no version is written to control panel, it would be best to include the apps and features entries